recipes/crispritz/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+make
+mkdir -p $PREFIX/bin
+chmod 700 -R *
+cp crispritz.py $PREFIX/bin
+cp buildTST $PREFIX/bin
+cp searchTST $PREFIX/bin
+cp searchBruteForce $PREFIX/bin
+cp -R sourceCode/Python_Scripts/ $PREFIX/bin
```

Comment:
Still need to move those things somewhere else. :)

Reply:
Sorry I forgot to change the file, now it's correct.

Reply:
> Still need to move those things somewhere else. :)

Hi, I updated the recipe and the build file.

Thanks.